example/linalg/CMakeLists.txt

```diff
@@ -23,5 +23,7 @@ ADD_EXAMPLE(lstsq2)
 ADD_EXAMPLE(solve1)
 ADD_EXAMPLE(solve2)
 ADD_EXAMPLE(solve3)
+ADD_EXAMPLE(svd1)
+ADD_EXAMPLE(svd2)
```

Comment:
```suggestion
ADD_EXAMPLE(svd)
ADD_EXAMPLE(svdvals)
```

Reply:
Neat idea, fixed in [b579d98](https://github.com/fortran-lang/stdlib/pull/808/commits/b579d9866cb51fb6b66a968c78b20a26dd88f083)